src/main/java/co/inventorsoft/jdbc/repository/MailDao.java

```diff
@@ -0,0 +1,12 @@
+package co.inventorsoft.jdbc.repository;
+
+import co.inventorsoft.jdbc.model.Mail;
+
+import java.util.List;
+
+public interface MailDao {
```

Comment:
Avoid DAO pattern.
You should work with Repository pattern (by the way existing code implements repository pattern, not DAO).